frameworks/elastic/docs/limitations.md

```diff
@@ -20,8 +20,13 @@ The maximum number of deployable nodes is constrained by the DC/OS cluster's res
 
 Upgrades and rolling configuration updates do not wait for a green status. During deployment and upgrades, the `serial` strategy does not wait for the Elastic service to reach green before proceeding to the next node.
 
-## Toggling TLS
 
-Toggling TLS requires doing a full-cluster restart. This is an [Elasticsearch limitation](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html). Since the service's default update strategy is a rolling upgrade (`serial`) changing its configuration to enable (or disable) transport encryption will result in nodes configured with TLS not being able to communicate with nodes configured with unencrypted networking (and vice-versa). A full-cluster restart is required (using the `parallel` update strategy). Make sure to have backups and plan for some downtime.
+## Security
+
+Elastic's native authentication and authorization mechanisms are not supported at this time.
```

Comment:
Should we clarify that "not supported" only means that it's not configurable from the service perspective? Authn and Authz are available and configurable when X-Pack is enabled.

Reply:
I'm going to leave it as the very broad "not supported" as we aren't supporting it directly, and the language leaves some wiggle room.